shesmu-server/src/test/java/ca/on/oicr/gsi/shesmu/UserInterfaceTest.java

```diff
@@ -38,7 +38,16 @@ public UserInterfaceTest() {
 
     final ChromeOptions opts = new ChromeOptions();
     opts.setHeadless(true);
-    opts.addArguments("--disable-gpu");
+    opts.addArguments("--window-size=1920,1080");
+    // I would love to tell you what these do, but without them, you get a nonspecific ChromeDriver
+    // error under Docker.
+    // https://stackoverflow.com/questions/50642308/webdriverexception-unknown-error-devtoolsactiveport-file-doesnt-exist-while-t/50642913#50642913
+    opts.addArguments("start-maximized"); // open Browser in maximized mode
+    opts.addArguments("disable-infobars"); // disabling infobars
```

Comment:
to confirm - these two don't use '--'?

Reply:
It was like that in the Stack Overflow. :man_shrugging: It doesn't produce any errors for me.